sys/arm64/conf/GENERIC-MORELLO

```diff
@@ -31,6 +31,8 @@ options		CHERI_CAPREVOKE_STATS
 options		CHERI_CAPREVOKE_FAST_COPYIN
 options		CHERI_CAPREVOKE_CLEARTAGS
 
+options		CHERI_C18N
```

Comment:
Why do we need an option for this?

Reply:
This is used in `sys/kern/imgact_elf.c` and `sys/kern/kern_procctl.c` to include code that allows you to use `procctl` to turn on compartmentalisation.

Reply:
I don’t think it needs to be an option

Reply:
That is, a CHERI-enabled arm64 kernel should always include this support IMO

Reply:
How do I always `#define` this without making it an option?

Reply:
I think you don't need this define. Just enable if the kernel is CHERI enabled and on Morello since it's harmless if rtld doesn't know about them. We'll want to get to the point where it's always on fairly quickly in DEC.